### PR TITLE
Add doc for device ID and Edge CA certs over EST

### PR DESCRIPTION
**Pull request by gordonwang0**
Describes how to configure aziot-edged to issue the device identity and Edge CA certificates over EST.

### CODE REVIEW

File: edgelet/doc/est.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
+# Configuring aziot-edged with EST
+
+This document covers how to configure aziot-edged to issue device identity and Edge CA certificates over EST.
```
Comment (micahl) [inline edgelet/doc/est.md:3]: Our public [Understand how IoT Edge uses certificates](https://docs.microsoft.com/en-us/azure/iot-edge/iot-edge-certs?view=iotedge-2020-11) describes a number of certificates.  Which one of those is the _Edge CA_ mentioned here?

@eustacea do the public docs on the certificates in IoT Edge need to be updated so we have consistent language everywhere?

File: edgelet/doc/est.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
+# Configuring aziot-edged with EST
```
Comment (micahl) [inline edgelet/doc/est.md:1]: Maybe re-title to something like ...
```suggestion
# Authenticate a Device Using Certificates Issued Dynamically via EST
```
?

**Review by micahl — CHANGES_REQUESTED** [review]

File: edgelet/doc/est.md (hunk 1 above)
Comment (gordonwang0) [reply edgelet/doc/est.md:3]: The documentation you linked is applies to 1.1 only (the note that it applies to 1.2 is incorrect).

In 1.2, the workload CA is gone. Server certificates are issued off the *Device CA cert* in the diagram, which we have renamed *Edge CA cert*.

Comment (micahl) [reply edgelet/doc/est.md:3]: Tagging @eustacea and @kgremban on the v1.2 doc for certificates needing updates

**Review by micahl — APPROVED** [review]

**Pull request merged**